runner.go

```diff
@@ -0,0 +1,67 @@
+package server
+
+import (
+	"net/http"
+	"net/url"
+	"time"
+)
+
+type Output struct {
```

Comment:
Won't `gob` complain ? By extension: ins't the name of the struct and it's field part of `gob` encoding ?

Reply:
It won't, only the type itself matter!

Reply:
I'm ok with either `RunResult` or `RunOutput`. No `Benckmark` because a benchmark is by definition a comparison of different implementation, which is not what the single run does. Hence `Run` is more appropriate to me.

Leaving `Benchmark` free for the future.

What do you prefer ?

Reply:
Let's go with `RunResult` then! Should we also rename `benchttp` to `runttp` meanwhile? 😛

Reply:
Arf f-it `Benchmark` is cool too and as a better readability:
- `benchttp.Benchmark.Report.Records`, `benchttp.Benchmark.Metadata.Config` vs
- `benchttp.RunResult.Report.Records`, `benchttp.RunResult.Metadata.Config`

Final call ?

Reply:
Haha, honestly both are fine to me! Maybe a little preference for the one that has `Result` in its name.

On a side note, I didn't put much thinking into the structure when doing `Metadata.Config`, could as well be `Meta.Config`, or even flattened `Output.Report`, `Output.Config`. It's probably the time to discuss the final shape of that output (including `*http.Request` and `*http.Response` objects): Discord?